lib/ui/myplot/my_plot_page.dart

```diff
@@ -2,10 +2,99 @@ import 'package:farmsmart_flutter/data/model/crop_entity.dart';
+  final double oppacityForDayCount;
+  final double separatorHeight;
+  final double sizeForDayCountShape;
+
+  HomeMyPlotPageStyle(this.iconButtonColor, this.primaryGreen,
+      this.separatorWhite,
+      this.edgeInsetsTop, this.edgeInsetsTitle, this.listPadding,
+      this.paddingBetweenElements,this.paddingBeforeCropDayCount,
+      this.paddingForDayCount,
+      this.ovalRadiousForDayCount,
+      this.circularProgressIndicatorAligmentCenter,
+      this.mainAxisAlignmentSpaceBeetwen, this.mainAxisAlignmentSpaceStart,
+      this.mainAxisSizeMin,
+      this.crossAxisAlignmentStart, this.titleTextStyle, this.cropNameTextStyle,
+      this.cropStatusTextStyle, this.cropDayTextStyle, this.buttonShapeRadious,
+      this.buttonIconSize, this.oppacityForDayCount, this.separatorHeight, this.separatorIndent,
+      this.sizeForDayCountShape);
+}
+
+class _DefaultStyle implements HomeMyPlotPageStyle {
+
+  final Color iconButtonColor =  const Color(0xFFFFFFFF);
+  final Color primaryGreen =  const Color(0xff25df0c);
+  final Color separatorWhite = const Color(0xfff5f8fa);
+
+  final EdgeInsets edgeInsetsTop = const EdgeInsets.only(top: 20.0) ;
+  final EdgeInsets edgeInsetsTitle = const EdgeInsets.only(left: 25, top: 30, right: 5, bottom: 20);
+  final EdgeInsets listPadding = const EdgeInsets.only(left: 25.0, top: 25.0, right: 30.0, bottom: 25.0);
+  final EdgeInsets paddingBetweenElements = const EdgeInsets.only(bottom: 30);
+
+  final EdgeInsets paddingBeforeCropDayCount = const EdgeInsets.only(bottom: 23);
+  final EdgeInsets paddingForDayCount = const EdgeInsets.only(left: 12, top: 8, right: 12, bottom: 5);
+  final EdgeInsets separatorIndent = const EdgeInsets.only(left: 25.0);
+
+  final BorderRadius ovalRadiousForDayCount = const BorderRadius.all(Radius.circular(20.0));
+
+  final Alignment circularProgressIndicatorAligmentCenter = Alignment.center;
+  final MainAxisAlignment mainAxisAlignmentSpaceBeetwen = MainAxisAlignment.spaceBetween;
+  final MainAxisAlignment mainAxisAlignmentSpaceStart = MainAxisAlignment.start;
+  final CrossAxisAlignment crossAxisAlignmentStart =   CrossAxisAlignment.start;
+  final MainAxisSize mainAxisSizeMin = MainAxisSize.min;
+
+  final TextStyle titleTextStyle = const TextStyle(fontSize: 27, fontWeight: FontWeight.bold, color: Color(0xFF000000));
+  final TextStyle cropNameTextStyle = const TextStyle(fontSize: 17, fontWeight: FontWeight.bold, color: Color(0xff1a1b46));
+  final TextStyle cropStatusTextStyle = const TextStyle(fontSize: 15, fontWeight: FontWeight.normal, color: Color(0xff767690));
+  final TextStyle cropDayTextStyle = const TextStyle(fontSize: 11, fontWeight: FontWeight.normal, color: Color(0xff25df0c));
+
+  final double buttonShapeRadious = 25.0;
+  final double buttonIconSize = 15.0;
+  final double oppacityForDayCount = 0.08;
+  final double separatorHeight = 2;
+  final double sizeForDayCountShape = 80.0;
+
+  const _DefaultStyle();
+}
 
 class HomeMyPlotPage extends StatefulWidget {
```

Comment:
Naming here could be just "PlotList"      "my" doesn't add much to context and "home" is a design detail (the plot list doesn't not have to be the home page)